packages/vue/src/components/FNavigationMenu/navigation-menu-item.ts

```diff
@@ -3,19 +3,19 @@
  */
 export interface NavigationMenuItem {
     /**
-     * Menu item label text
+     * Text to display for the menu item.
      */
     label: string;
     /**
-     * Menu item route used upon for example item selection
+     * Unique route/path for the menu item. Used for item selection and reference.
```

Comment:
Som läsare är jag nog lite förvirrad över denna. Exemplen har varit lite tvetydiga, jag vet inte vad jag ska skicka in här.

Om jag använder vue-router:

* Är det `path` parametern från rutten? `#/foobar`? `/foobar`? `foobar`? Ska det matcha `path` i rutten?
* Är det namned på rutten? Dvs ska det matcha `name` i rutten?

Om jag inte använder vue-router:

* Då måste jag väl sätta `href` eller? Vad har jag denna till? Är det bara en unik identifierare?